server/seeders/seeds.js

```diff
@@ -1,120 +1,89 @@
+    const rating = faker.finance.amount();
+    //(min?: 1, max?: 10)
+    const voteCount = faker.random.number();
+    const title = faker.commerce.productName();
+    const overview = faker.lorem.words(Math.round(Math.random() * 20) + 1);
+    const releaseDate = faker.date.past();
+    const poster = faker.image.imageUrl();
+    const trailer = faker.image.imageUrl();
+    //example//'https://image.tmdb.org/t/p/w185_and_h278_bestv2/rplLJ2hPcOQmkFhTqUte0MkEaO2.jpg'
+//externalMovieId, rating, voteCount, title, overview, releaseDate, poster, trailer
 
     const randomUserIndex = Math.floor(Math.random() * createdUsers.ops.length);
     const { username, _id: userId } = createdUsers.ops[randomUserIndex];
 
-    const createdComment = await Comment.create({ commentText, username });
+    const likedMovie = await Movie.create({ externalMovieId, rating, voteCount, title, overview, releaseDate, poster, trailer, username });
 
     const updatedUser = await User.updateOne(
       { _id: userId },
-      { $push: { comments: createdComment._id } }
+      { $push: { movies: likedMovie._id } }
     );
 
-    createdComments.push(createdComment);
+    likedMovies.push(likedMovie);
   }
 
-  // create reactions
-  for (let i = 0; i < 100; i += 1) {
-    const reactionBody = faker.lorem.words(Math.round(Math.random() * 20) + 1);
-
-    const randomUserIndex = Math.floor(Math.random() * createdUsers.ops.length);
-    const { username } = createdUsers.ops[randomUserIndex];
-
-    const randomCommentIndex = Math.floor(Math.random() * createdComments.length);
-    const { _id: commentId } = createdComments[randomCommentIndex];
-
-    await Comment.updateOne(
-      { _id: commentId },
-      { $push: { reactions: { reactionBody, username } } },
-      { runValidators: true }
-    );
-  }
+   // create dislikedMovies
+   let dislikedMovies = [];
+   for (let i = 0; i < 100; i += 1) {
+     const externalMovieId = faker.random.number();
+     const rating = faker.finance.amount();
+     //(min?: 1, max?: 10)
+     const voteCount = faker.random.number();
+     const title = faker.commerce.productName();
+     const overview = faker.lorem.words(Math.round(Math.random() * 20) + 1);
+     const releaseDate = faker.date.past();
+     const poster = faker.image.imageUrl();
+     const trailer = faker.image.imageUrl();
+     //example//'https://image.tmdb.org/t/p/w185_and_h278_bestv2/rplLJ2hPcOQmkFhTqUte0MkEaO2.jpg'
+ //externalMovieId, rating, voteCount, title, overview, releaseDate, poster, trailer
+ 
+     const randomUserIndex = Math.floor(Math.random() * createdUsers.ops.length);
+     const { username, _id: userId } = createdUsers.ops[randomUserIndex];
+ 
+     const dislikedMovie = await Movie.create({ externalMovieId, rating, voteCount, title, overview, releaseDate, poster, trailer, username });
+ 
+     const updatedUser = await User.updateOne(
+       { _id: userId },
+       { $push: { movies: dislikedMovie._id } }
```

Comment:
similarly, should this be `{ $addToSet: { dislikedMovies: likedMovie._id } }`?

Reply:
not quite, but I have refactored the code differently so you have likedMovies and dislikedMovies